upgrade-2.x.sh

```diff
@@ -787,8 +787,12 @@ fi
 # Log timer
 starttime=$(date +%s)
 
+# For compatibility purposes
```

Comment:
@saintaardvark leaving this around in case we rely on this somehow (or customers do)